portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -45,6 +45,16 @@ definition {
 			value1 = "Displaying Form: ${formName} (Modified)");
 	}
 
+	macro selectFormOnAPage {
```

Comment:
[This](https://github.com/liferay-bpm-qa/liferay-portal/pull/1108#issuecomment-1717955219) improvement is being implemented here